### ISSUE TITLE
Fix update checking

### ISSUE THREAD

**Issue by Sanaki** [victor-rds/KeeTrayTOTP#16]
Having glanced through the code, it appears making update checks work would be as simple as updating  KeeTrayTOTP/TrayTOTP_Plugin.cs line 842 to point at KeeTrayTOTP/version_manifest.txt and updating the version number there with each release.

**Comment by robinvanpoppel** [comment]
@Sanaki, thanks for reporting this. I just pushed a commit to let it point to the right version_manifest.txt and current version (0.9.1), but I'll ultimately have to create a release to make this work.

**Comment by Sanaki** [comment]
Of course. I just figured it got overlooked. Thanks for the quick fix.
